cli/build.rs

```diff
@@ -1,17 +1,38 @@
-use std::path::PathBuf;
+use std::path::{Path, PathBuf};
 use std::{env, fs};
+use std::vec::Vec;
 
 fn main() {
     if let Some(git_sha) = read_git_sha() {
         println!("cargo:rustc-env={}={}", "BUILD_SHA", git_sha);
     }
 
+    if wasm_files_present() {
+        println!("cargo:rustc-cfg={}", "TREE_SITTER_EMBED_WASM_BINDING");
+    }
+
     println!(
         "cargo:rustc-env=BUILD_TARGET={}",
         std::env::var("TARGET").unwrap()
     );
 }
 
+fn required_files() -> Vec<&'static Path> {
+    return vec![
+        Path::new("../lib/binding_web/tree-sitter.js"),
+        Path::new("../lib/binding_web/tree-sitter.wasm")
+        ];
```

Comment:
Since this method  now only has one definition, I would probably just inline it into `wasm_files_present`.